Interfaces/API/DiracAdmin.py

```diff
@@ -262,9 +262,11 @@ def getSiteMaskLogging(self, site=None, printOutput=False):
             gLogger.notice(stup)
           gLogger.notice(' ')
       elif isinstance(sitesLogging, list):
-        result = [(sl[1], sl[3], sl[4]) for sl in sitesLogging]
+	sitesLoggingList = [(sl[1], sl[3], sl[4]) for sl in sitesLogging]
+	for siteLog in sitesLoggingList:
+	  gLogger.notice(siteLog)
 
-    return result
+    return S_OK()
```

Comment:
Is this right? I haven't looked properly but I was expecting it to return the result of line 241 or 243.

Reply:
That's returned at line 246 if it's `S_ERROR`, otherwise if it reaches here it's only the printout.